content/docs/rfcs/64/README.md

```diff
@@ -0,0 +1,258 @@
+1. [12/WAKU2-FILTER](https://rfc.vac.dev/spec/12/) to allow resource-restricted peers to subscribe to messages matching a specific content filter.
+2. [13/WAKU2-STORE](https://rfc.vac.dev/spec/13/) to allow other peers to request historical messages from this node. The store SHOULD be configured to retain at least `12` hours of messages per supported shard.
+3. [19/WAKU2-LIGHTPUSH](https://rfc.vac.dev/spec/19/) to allow resource-restricted peers to request publishing a message to the network on their behalf.
+4. [34/WAKU2-PEER-EXCHANGE](https://rfc.vac.dev/spec/34/) to allow resource-restricted peers to discover more peers in a resource efficient way.
+
+#### Non-relay nodes
+
+Nodes MAY opt out of relay functionality on any network shard
+and instead request services from relay nodes as clients
+using any of the defined service protocols:
+1. [12/WAKU2-FILTER](https://rfc.vac.dev/spec/12/) to subscribe to messages matching a specific content filter.
+2. [13/WAKU2-STORE](https://rfc.vac.dev/spec/13/) to request historical messages matching a specific content filter.
+3. [19/WAKU2-LIGHTPUSH](https://rfc.vac.dev/spec/19/) to request publishing a message to the network.
+4. [34/WAKU2-PEER-EXCHANGE](https://rfc.vac.dev/spec/34/) to discover more peers in a resource efficient way.
+
+### Applications
+
+Applications are the higher-layer projects or platforms that make use of the generalized messaging capability of the network.
+In other words, an application defines a payload used in the various [10/WAKU2](https://rfc.vac.dev/spec/10/) protocols.
+Any participant in an application SHOULD make use of an underlying node in order to communicate on the network.
+Applications SHOULD make use of an [autosharding](#autosharding) API
+to allow the underlying node to automatically select the target shard on the Waku Network. See the section on [autosharding](#autosharding) for more.
+
+## RLN rate-limiting
+
+The [17/WAKU2-RLN-RELAY](https://rfc.vac.dev/spec/17/) network uses [32/RLN-V1](https://rfc.vac.dev/spec/32/) proofs
+to ensure that a pre-agreed rate limit is not exceeded by any publisher.
+While the network is under capacity,
+individual relayers MAY choose to freely route messages without RLN proofs
+up to a discretionary bandwidth limit after which messages without proofs MUST be discarded.
+This implies that quality of service and reliability is significantly lower for messages without proofs
+and at times of high network utilization these messages may not be relayed at all.
+
+### RLN Parameters
+
+For the Waku Network,
+the `epoch` is set to `1` second
+and the maximum number of messages published per `epoch` is limited to `1` per publisher.
+The `max_epoch_gap` is set to `20` seconds,
+meaning that validators MUST _ignore_ messages with an `epoch` more than 20 seconds into the past or future compared to the validator's own clock.
+
+### Memberships
+
+Each publisher to the Waku Network SHOULD register an RLN membership
```

Comment:
The main (only?) way of rate limiting in the network is RLN and for that you need a membership. The alternative is the free tier, but we can't rate limit publishers here as the messages are not linked to their origin. In other words, a relay node doesn't have any idea as to the origin of the messages it's routing.

Reply:
Okay got it, thanks!
Then, we will need to strive to have a very simple registration process, and in there, give the option to potentially register a free-tier account, so that anyone can use the network freely. wdyt?